api/logging_api/routes/LedSign.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const axios = require('axios');
+const router = express.Router();
+const { LED_SIGN_URL } = require('../../config/config.js');
```

Comment:
2 things:
- [ ] merge this line with line 7 since you are already importing from config.json
- [ ] `LED_SIGN_URL` is not defined in config.example.json, you should add this